plugins/MobileMessaging/Controller.php

```diff
@@ -155,4 +158,32 @@ public function getCredentialFields()
 
         return $view->render();
     }
+
+    public function getBrowserNotifications()
```

Comment:
Can we move this into an API method please? In the request URL you then maybe need to append `&format=json`.
This allows other users to use the token as well and embed notifications into other apps using other formats etc.